source/kraken/fill_disruption_from_database.h

```diff
@@ -252,5 +264,28 @@ namespace navitia {
             FILL_NULLABLE(channel, max_size, uint32_t)
         }
 
+        template<typename T>
+        void fill_channel_type(T const_it, chaos::Channel* channel) {
+            const auto& type_ = const_it["channel_type"].template as<std::string>();
+            //Here we have to verify existance of Channel Type in the channel.
+            if (type_ == "web"){
```

Comment:
just for the consistence, can you use switch like you've done here https://github.com/CanalTP/navitia/pull/1168/files#diff-e609fca02cbf3693aa5cfeffb8a75685R213 ?